frontend/package.json

```diff
@@ -30,13 +30,14 @@
     "validator": "^13.9.0"
   },
   "scripts": {
-    "start": "vite",
     "build": "tsc && vite build",
+    "format": "prettier --write \"src/**/*.ts\"",
```

Comment:
I'm curious about whether js and tsx files are formatted as well? or scss files

Reply:
opted to just include `.ts` and `.tsx` files since we don't have any `.js` files in our src folder. Not sure if we want to include `.scss` files...but I'll ask